README.adoc

```diff
@@ -198,17 +172,25 @@ include::{common-includes}/mvnbuild.adoc[]
 
 When the server is running, point your browser to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-You receive a result in JSON format with the system properties of your local JVM. Next, point your
-browser to the `http://localhost:9080/system/properties` URL, which retrieves the information for a
-specific host. Notice that the results from the two URLs are identical.
+You receive the system properties of your local JVM from the `inventory` service. Next, point your
+browser to the the `system` service URL, which is `http://localhost:9080/system/properties`,
```

Comment:
Please remove the duplicate "the" on line 176.

Reply:
I usually don't like URLs like `http://localhost:9080/system/properties` to stand alone, but "the `http://localhost:9080/system/properties` `system` service URL" looks confusing, almost as if "system" were part of the URL, too.

What about "...,which is located at `http://localhost:9080/system/properties`,"

Reply:
Fixed